components/pictogram/tests/unit/pictograms.spec.tsx

```diff
@@ -0,0 +1,72 @@
+import { GEL } from '@westpac/core';
+import { render } from '@testing-library/react';
+import wbc from '@westpac/wbc';
+import { overridesTest } from '../../../../helpers/tests/overrides-test.js';
+import { nestingTest } from '../../../../helpers/tests/nesting-test.js';
+import * as pictograms from '@westpac/pictogram';
+import { Pictogram, PictogramProps } from '../../src/Pictogram.tsx';
+
+overridesTest({
+	name: 'pictogram',
+	overrides: ['Pictogram'],
+	Component: (props: any) => <Pictogram copyrightYear="2023" {...props} />,
+});
+
+nestingTest({
+	name: 'pictogram',
+	Component: (props: any) => <Pictogram {...props} />,
+});
```

Comment:
Just curious to know how the Pictogram renders without throwing a warning about missing the copyrightYear prop.

Reply:
Discussed on Teams

Reply:
From Teams: Spreading the props in the nesting test will make it use the default value for required props if they exist.